huaweicloud/services/fgs/resource_huaweicloud_fgs_async_invoke_configuration.go

```diff
@@ -0,0 +1,221 @@
+package fgs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/fgs/v2/function"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceAsyncInvokeConfiguration() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAsyncInvokeConfigurationCreate,
+		ReadContext:   resourceAsyncInvokeConfigurationRead,
+		UpdateContext: resourceAsyncInvokeConfigurationUpdate,
+		DeleteContext: resourceAsyncInvokeConfigurationDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceAsyncInvokeConfigImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "he region in which to configure the asynchronous invocation.",
```

Comment:
he region in which to configure the asynchronous invocation.
The region in which to configure the asynchronous invocation.

Reply:
done.